src/test/java/uk/gov/cslearning/catalogue/api/CourseControllerTest.java

```diff
@@ -314,28 +336,28 @@ public void shouldGetRequiredLearningByOrgCodeMapWithParams() throws Exception {
                 .andExpect(status().isOk());
     }
 
-    @Test
```

Comment:
why has this test been deleted?  If there is a valid reason / not applicable anymore then far enough!